.github/workflows/release-2-stabilize.yml

```diff
@@ -0,0 +1,131 @@
+name: 'CD: stabilize'
+
+on:
+  push:
+    branches: [ 'release/**' ]
+
+jobs:
+
+  changes:
+    runs-on: ubuntu-22.04
+    permissions: { pull-requests: read }
+    steps:
+      - uses: actions/checkout@v4
+      - id:   filter
+        uses: dorny/paths-filter@v3
+        name: Detect changed files
+        with: { filters: .github/filters.yml }
+    outputs:
+      darker:   ${{ steps.filter.outputs.darker   }}
+      pytest:   ${{ steps.filter.outputs.pytest   }}
+      npm-test: ${{ steps.filter.outputs.npm-test }}
+
+  darker:
+    needs: changes
+    if: needs.changes.outputs.darker == 'true'
+    uses: ./.github/workflows/darker.yml
+
+  pytest:
+    needs: changes
+    if: needs.changes.outputs.pytest == 'true'
+    uses: ./.github/workflows/pytest.yml
+
+  npm-test:
+    needs: changes
+    if: needs.changes.outputs.npm-test == 'true'
+    uses: ./.github/workflows/npm-test.yml
+
+  version:
+    runs-on: ubuntu-24.04
+    outputs:
+      version: ${{ steps.version.outputs.version }}
+    steps:
+    - uses: actions/checkout@v4
+    - name: Formats the version number
+      id: version
+      run: |
+        set -xe
+        BRANCH=git rev-parse --abbrev-ref HEAD
+        VERSION=`echo $BRANCH | sed -e 's/release\///'`
+        echo "version=$VERSION" >> "$GITHUB_OUTPUT"
+
+  changelog:
+    runs-on: ubuntu-24.04
+    steps:
+    - name: post to Zulip Linear updated draft of changelog
+      run: |
+        echo 'TODO, for now generate changelog manually'
+        echo 'TODO, for now send to Zulip manually'
+
+  deploy-to-beta:
+    runs-on: ubuntu-22.04
```

Comment:
same question? Why do we mix both versions of ubuntu? any problem to keep `24.04` everywhere?